src/TLuaInterpreter.cpp

```diff
@@ -3000,26 +3000,26 @@ int TLuaInterpreter::setBackgroundImage(lua_State* L)
     return 0;
 }
 
-int TLuaInterpreter::setLabelClickCallback(lua_State* L)
+int TLuaInterpreter::setLabelCallback(lua_State* L, QString funcName)
 {
     Host& host = getHostFromLua(L);
 
     QString labelName;
     if (!lua_isstring(L, 1)) {
-        lua_pushfstring(L, "setLabelClickCallback: bad argument #1 type (label name as string expected, got %s!)", luaL_typename(L, 1));
+        lua_pushfstring(L, "(%s: bad argument #1 type (label name as string expected, got %s!))", funcName.toUtf8().constData(), luaL_typename(L, 1));
         return lua_error(L);
     } else {
         labelName = QString::fromUtf8(lua_tostring(L, 1));
         if (labelName.isEmpty()) {
             lua_pushnil(L);
-            lua_pushstring(L, "setLabelClickCallback: bad argument #1 value (label name cannot be an empty string.)");
+            lua_pushfstring(L, R"(%s: bad argument #1 value (label name cannot be an empty string.))", funcName.toUtf8().constData());
```

Comment:
Be careful with C++11 delimited raw string literals - there are suspicions (on my part and others https://bugreports.qt.io/browse/QTBUG-42736) that they screw around with the processing of any implementation unit (class definition / .cpp) file which contain texts that will need to be extracted for translation which is something I am working on (#1334) for Mudlet 4.0 .  If there are any `tr("Text that will be translated")` - and I think there are - in this file we may have to refrain from using this useful addition to C++ standard because Qt's `lupdate` utility which gathers the texts breaks when it encounters them, even when they are not being used to define a translatable string.

{I believe that these constructs, being able to be have more than one line of text in them and which will embed the line feeds so included, do not pass through the parsing system that lupdate uses and jam it up like a literal spanner in the works!}